test/utilities/JsonSpec.scala

```diff
@@ -12,39 +13,17 @@ class JsonSpec
 
   "toText" should {
     "return Json String " in {
-      val testObj = Message("wisa", "Test Message")
+      val testObj = MessageMock.messageForm
 
-      testObj.toText must include("wisa")
+      testObj.toText must include("6010545811")
```

Comment:
Test with object's attribute instead of hardcoded test case

```suggestion
      testObj.toText must include testObj.id
```